curriculum/utils.js

```diff
@@ -16,3 +16,33 @@ exports.testedLang = function testedLang() {
     throw Error('LOCALE must be set for testing');
   }
 };
+
+// TODO: migrate to TS and use the SuperBlocks enum from
+// config/certification-settings.ts
+
+const superBlockToOrder = {
+  'responsive-web-design': 0,
+  'javascript-algorithms-and-data-structures': 1,
+  'front-end-development-libraries': 2,
+  'data-visualization': 3,
+  'relational-databases': 4,
+  'back-end-development-and-apis': 5,
+  'quality-assurance': 6,
+  'scientific-computing-with-python': 7,
+  'data-analysis-with-python': 8,
+  'information-security': 9,
+  'machine-learning-with-python': 10,
+  'coding-interview-prep': 11,
+  certifications: null
```

Comment:
So I think we can revert all of this logic now that certifications don't have a meta.

Reply:
Just drop the 'certifications' special cases? Great, that'll be much nicer.

Reply:
Right because we no longer have certification metas at all, right?

Reply:
Sorry, just saw this. Yes, exactly. 

Somehow despite working on both PRs I didn't put 2 and 2 together...